packages/prices/utils/price.ts

```diff
@@ -120,7 +120,7 @@ export const getCurrency = (
 export const formatPrice = (
 	// Price in minor unit, e.g. cents.
 	price: number | string,
-	currencyData: Currency
+	currencyData?: Currency
```

Comment:
Good catch with this. I noticed there were several other instances where we call `formatPrice` without the second argument.